dubbo-common/src/main/java/org/apache/dubbo/common/logger/slf4j/Slf4jLoggerAdapter.java

```diff
@@ -45,6 +52,7 @@ public Level getLevel() {
 
     @Override
     public void setLevel(Level level) {
+        getLogger(Slf4jLoggerAdapter.class).warn(String.format("The level of slf4j logger can not be set, using the default level: %s",Slf4jLogger.getLevel(ROOT_LOGGER)));
```

Comment:
Please follow https://dubbo.apache.org/en/docs3-v2/java-sdk/faq/intro/ to configure logger code

Reply:
hello,I just fix,can you take a look?